lib/WebpackError.js

```diff
@@ -4,8 +4,81 @@
 */
 "use strict";
 
+const createHash = require("./util/createHash");
+
+const hash = Symbol();
+const meta = Symbol();
+
 module.exports = class WebpackError extends Error {
+	constructor(
+		message,
+		line = 0,
```

Comment:
`lineEnd` would make sense for multiple line pointers.

The parser uses location objects in this format: `{ start: { line, column }, end: { line, column } }`. Does it make sense to use the same format here instead of multiple arguments?

Reply:
I have no issue with using the same format. The only reason I chose this format is that it happens to be the same that Firefox uses. Otherwise, there is no standard for this.